evap/evaluation/tests/test_views.py

```diff
@@ -48,3 +52,12 @@ class TestLegalNoticeView(ViewTest):
 class TestFAQView(ViewTest):
     url = '/faq'
     test_users = ['']
+
+
+class TestSendFeedbackView(WebTest):
+    csrf_checks = False
+
+    def test_sends_mail(self):
+        with patch('django.core.mail.EmailMessage.send') as mock:
+            self.app.post('/feedback/send', {'message': 'feedback message', 'sender_email': 'unique@mail.de'}, user='')
+        mock.assert_called_once_with(False)
```

Comment:
put your hammer out of your hand for a sec and use [`mail.outbox`](https://docs.djangoproject.com/ja/1.9/topics/testing/tools/#email-services) :)